homeassistant/components/wink.py

```diff
@@ -201,11 +201,19 @@ def _get_wink_token_from_web():
             return False
         pywink.set_bearer_token(token)
 
-    client_id = config[DOMAIN].get(ATTR_CLIENT_ID)
-    client_secret = config[DOMAIN].get(ATTR_CLIENT_SECRET)
-    email = config[DOMAIN].get(CONF_EMAIL)
-    password = config[DOMAIN].get(CONF_PASSWORD)
-    local_control = config[DOMAIN].get(CONF_LOCAL_CONTROL)
+    try:
+        client_id = config[DOMAIN].get(ATTR_CLIENT_ID)
+        client_secret = config[DOMAIN].get(ATTR_CLIENT_SECRET)
+        email = config[DOMAIN].get(CONF_EMAIL)
+        password = config[DOMAIN].get(CONF_PASSWORD)
+        local_control = config[DOMAIN].get(CONF_LOCAL_CONTROL)
+    except KeyError:
```

Comment:
You will only get a KeyError if `wink` does not exist in config. If that is what you want to check, please check for that explicitly.